homeassistant/components/co2signal/sensor.py

```diff
@@ -77,7 +78,7 @@ def __init__(
         self.entity_description = description
 
         self._attr_extra_state_attributes = {
-            "country_code": coordinator.data.country_code,
+            CONF_COUNTRY_CODE: coordinator.data.country_code,
```

Comment:
It would be nicer if this was named `ATTR_` instead of `CONF_` as I think its mixing concepts here

Reply:
Yea, just used the const from HA itself as in the config flow but could change this